lib/commands/clone.js

```diff
@@ -245,7 +245,7 @@ module.exports = function *(argv) {
     })
 
     yield asyncEach(tablesToCopyList, function *(table, idx) {
-      let cursor = yield sr.db(sourceDB).table(table).run({cursor: true})
+      let cursor = yield sr.db(sourceDB).table(table).run({cursor: true, timeFormat: 'raw'})
```

Comment:
My bad, was thinking about a different part of the file when I read this.

Ignore prior comment.